### PR TITLE
Add box drawing support

### PR DESCRIPTION
**Pull request by mgazza**

### CODE REVIEW

**Comment by andydotxyz** [comment]
This looks really cool thanks. But like the other it has conflicts that need resolved before its tests can run.

**Comment by andydotxyz** [comment]
Some of the added tests are failing as you can see in the actions report...

**Comment by mgazza** [comment]
OK thats now sorted. I failed when rebasing

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Please note failing static check reporting missing docs

**Comment by andydotxyz** [comment]
Seems to work well btw, thanks for this - just get `staticcheck` passing and we're good.

File: output.go [hunk 1]
```diff
@@ -199,3 +257,11 @@ func handleOutputTab(t *Terminal) {
 		t.handleOutputChar(' ')
 	}
 }
+
+func handleShitOut(t *Terminal) {
+	t.useG1CharSet = true
+}
+
+func handleShitIn(t *Terminal) {
```
Comment (Jacalz) [inline output.go:265]: Should these really contain "shit"?

**Review by andydotxyz — APPROVED** [review]
Thanks!

**Pull request merged**